spec/spec_helper.rb

```diff
@@ -1,4 +1,8 @@
 require "coveralls"
-require "rspec"
-require "fakefs"
 require "crumple"
+require 'rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.